apps/ui/src/routes/changelog/index.tsx

```
@@ -0,0 +1,46 @@
+			{ name: "twitter:title", content: "Changelog - LLM Gateway" },
+			{
+				name: "twitter:description",
+				content:
+					"Stay up to date with the latest features, improvements, and fixes in LLM Gateway.",
+			},
+		],
+	}),
+	component: Changelog,
+});
+
+function Changelog() {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Changelog'. Consider to delete it or rename it.

'Changelog' is defined here:


(lint/suspicious/noRedeclare)